internal/builders/generic/attest.go

```diff
@@ -132,10 +138,13 @@ run in the context of a Github Actions workflow.`,
 		&attPath, "signature", "g", "",
 		"Path to write the signed provenance.",
 	)
+	c.Flags().StringVarP(
+		&subjectsFilename, "subjects-filename", "f", "",
+		"Filename containing a formatted list of subjects in the same format as sha256sum (base64 encoded).",
+	)
 	c.Flags().StringVarP(
 		&subjects, "subjects", "s", "",
```

Comment:
Do we even use the subjects flag anymore? Can we remove it?

Reply:
+1